### ISSUE TITLE
Conversion fails due to incompatible package install?

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/staged-recipes#8628]
Appears that conversion [fails due to attempting to install incompatible packages]( https://travis-ci.org/conda-forge/staged-recipes/builds/549897290#L823 ). It seemed to be related to the `openssl` version installed, but fixing the `openssl` version [didn't seem to work]( https://github.com/conda-forge/staged-recipes/pull/8626#issuecomment-505166829 ). Not sure what else might be causing it.

cc @conda-forge/core

**Comment by jakirkham** [comment]
It's worth noting that conversion [worked previously with Conda 4.7.4]( https://travis-ci.org/conda-forge/staged-recipes/builds/549890889#L881 ). So that doesn't seem like the issue.

**Comment by jjhelmus** [comment]
The python 3.7.3 package that is found requires openssl >=1.1.1b but the specification given on the install command explicitly require openssl 1.1.1a.  Can that spec be dropped from the install line?
```
pkgs/main/linux-64::python==3.7.3=h0371630_0 -> openssl[version='>=1.1.1b,<1.1.2a']
```

**Comment by jakirkham** [comment]
So that should have been solved with PR ( https://github.com/conda-forge/staged-recipes/pull/8626 ) as [seen here]( https://travis-ci.org/conda-forge/staged-recipes/builds/549921570#L884-L885 ). Unfortunately it still runs into [installation issues later]( https://travis-ci.org/conda-forge/staged-recipes/builds/549921570#L945 ). Though I may be missing something else.

**Comment by jakirkham** [comment]
Strange. Seems [this build]( https://travis-ci.org/conda-forge/staged-recipes/builds/549937947 ) didn't run into the same problem. Not sure what changed.

**Comment by jakirkham** [comment]
Though [this build]( https://travis-ci.org/conda-forge/staged-recipes/builds/549958533 ) is now back to the original problem. 😕

**Comment by jakirkham** [comment]
Apparently [just retrying made the build work]( https://travis-ci.org/conda-forge/staged-recipes/builds/549959938 ). Something really strange is going on here.

**Comment by jakirkham** [comment]
Just for vis, [still seeing this issue]( https://travis-ci.org/conda-forge/staged-recipes/builds/550327416 ).

**Comment by jakirkham** [comment]
Looks like all builds today failed [until this one]( https://travis-ci.org/conda-forge/staged-recipes/builds/550515345 ). Seems the failures are becoming more frequent.

**Comment by jakirkham** [comment]
@jjhelmus, I'm not sure I follow your suggestion. [`openssl` 1.1.1b is already included]( https://travis-ci.org/conda-forge/staged-recipes/builds/550742836#L462 ) when this failure occurs. What am I missing?

**Comment by jakirkham** [comment]
Is it possible that Conda does not think `1.1.1b` is within `>=1.1.1a,<1.1.2a`?

**Comment by scopatz** [comment]
Thanks for looking into this @jakirkham

**Comment by jjhelmus** [comment]
> I'm not sure I follow your suggestion. [`openssl` 1.1.1b is already included](https://travis-ci.org/conda-forge/staged-recipes/builds/550742836#L462) when this failure occurs. What am I missing?

I was thinking of when openssl was pinned to 1.1.1a in #8626.  This would prevent an update to 1.1.1b which is required for Python 3.7.3

**Comment by jakirkham** [comment]
Yeah I tried that in response to [update failing when we still had `openssl` 1.1.1b included]( https://travis-ci.org/conda-forge/staged-recipes/builds/549908215#L823 ) thinking it might help. This still happens [even when forcing `openssl` 1.1.1b]( https://travis-ci.org/conda-forge/staged-recipes/builds/550854194#L823 ).

**Comment by jakirkham** [comment]
Alright I've tried to make things work with Conda 4.7, but it doesn't seem to be working. Am proposing we just pin to Conda 4.6. ( https://github.com/conda-forge/staged-recipes/pull/8650 ) Please merge that PR if you agree.

**Comment by jakirkham** [comment]
Have raised upstream issue ( https://github.com/conda/conda/issues/8828 ).

Let's leave this issue open until we've seen conversion work reliably for a bit.
